app/components/UI/Ramp/common/components/ErrorView.tsx

```diff
@@ -96,14 +96,15 @@ function ErrorView({
     selectedRegion,
     selectedAsset,
     selectedFiatCurrencyId,
+    isBuy,
   } = useRampSDK();
 
   const ctaOnPressCallback = useCallback(() => {
     ctaOnPress?.();
   }, [ctaOnPress]);
 
   useEffect(() => {
-    trackEvent('ONRAMP_ERROR', {
+    trackEvent(`${isBuy ? 'ON' : 'OFF'}RAMP_ERROR`, {
```

Comment:
This is correct but a little bit too complex for the future, let's change it to the approach seen in other files 
```suggestion
    trackEvent(isBuy ? 'ONRAMP_ERROR' : 'OFFRAMP_ERROR', {
```